show/platform.py

```diff
@@ -5,20 +5,40 @@
 import click
 import utilities_common.cli as clicommon
 from sonic_py_common import device_info, multi_asic
+from swsscommon import swsscommon
 
 
+#
+# Constants
+# 
+
+CHASSIS_INFO_TABLE = 'CHASSIS_INFO|chassis {}'
+CHASSIS_INFO_CARD_NUM_FIELD = 'module_num'
+CHASSIS_INFO_SERIAL_FIELD = 'serial'
+CHASSIS_INFO_MODEL_FIELD = 'model'
+CHASSIS_INFO_REV_FIELD = 'revision'
+
+# Get hardware information
 def get_hw_info_dict():
     """
     This function is used to get the HW info helper function
     """
     hw_info_dict = {}
 
+    # Init statedb connection
+    db = swsscommon.SonicV2Connector()
```

Comment:
Is it possible to add some fallback way in the case when DB is not available but we still want these info? some of the info can be decoded from the EEPROM when DB is not available.

Reply:
I'm sure its possible, do you have any examples of where this is already implemented? 

If this isn't anywhere I am glad to add something that does this but I would prefer to do it as a separate pull request because I think it would take a little high level design work to come to the best solution that minimizes code reuse.

Reply:
like this: serial_number_cmd = "sudo decode-syseeprom -s",   when DB is not available, decode-syseeprom will decode the eeprom directly.